<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MemoryScannerWindow</class>
 <widget class="QWidget" name="MemoryScannerWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>833</width>
    <height>610</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Memory Scanner</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/icons/duck.png</normaloff>:/icons/duck.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QTableWidget" name="scanTable">
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SelectionMode::ContiguousSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectionBehavior::SelectRows</enum>
     </property>
     <property name="showGrid">
      <bool>false</bool>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <attribute name="horizontalHeaderHighlightSections">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>Address</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Value</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Previous Value</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Search Parameters</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Value:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLineEdit" name="scanValue"/>
        </item>
        <item>
         <widget class="QComboBox" name="scanValueSigned">
          <property name="currentIndex">
           <number>1</number>
          </property>
          <item>
           <property name="text">
            <string>Signed</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Unsigned</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="scanValueBase">
          <property name="currentIndex">
           <number>1</number>
          </property>
          <item>
           <property name="text">
            <string>Decimal</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Hex</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Data Size:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="scanSize">
        <property name="currentIndex">
         <number>1</number>
        </property>
        <item>
         <property name="text">
          <string>Byte (1 byte)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Halfword (2 bytes)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Word (4 bytes)</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Operator:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QComboBox" name="scanOperator">
        <item>
         <property name="text">
          <string>Any Value ('New Search' or 'Reset Result Value')</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Less Than Previous Result</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Less or Equal to Previous Result</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Greater Than Previous Result</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Greater or Equal to Previous Result</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Not Equal to Previous Result (Changed Value)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Equal to Previous Result (Unchanged Value)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Decreased By Value</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Increased By Value</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Changed By Value</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Equal to Value</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Not Equal to Value</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Less Than Value</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Less or Equal to Value</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Greater Than Value</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Greater or Equal to Value</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Start Address:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="scanStartAddress"/>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>End Address:</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLineEdit" name="scanEndAddress"/>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Preset Range:</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QComboBox" name="scanPresetRange">
        <item>
         <property name="text">
          <string>RAM</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Scratchpad</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>BIOS</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Result Count:</string>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QLabel" name="scanResultCount">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="9" column="0" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QPushButton" name="scanNewSearch">
          <property name="text">
           <string>New Search</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="scanSearchAgain">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Search Again</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="scanResetSearch">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Clear Results</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="10" column="0" colspan="2">
       <widget class="QPushButton" name="scanAddWatch">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Add Selected Results To Watch List</string>
        </property>
       </widget>
      </item>
      <item row="11" column="0" colspan="2">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="0" column="0">
      <widget class="QTableWidget" name="watchTable">
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::SelectionMode::ContiguousSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectionBehavior::SelectRows</enum>
       </property>
       <property name="showGrid">
        <bool>false</bool>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
       <attribute name="horizontalHeaderHighlightSections">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <column>
        <property name="text">
         <string>Simple Cheat Code or Description</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Address</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Type</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Value</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Freeze</string>
        </property>
       </column>
      </widget>
     </item>
     <item row="1" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QPushButton" name="scanLoadWatch">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="visible">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Load Watch</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="scanSaveWatch">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="visible">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Save Watch</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="scanAddManualAddress">
         <property name="text">
          <string>Add Manual Address</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="scanFreezeWatch">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Freeze Selected Entries</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="scanRemoveWatch">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Remove Selected Entries</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="cheatEngineAddress">
         <property name="frameShape">
          <enum>QFrame::Shape::Panel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Shadow::Sunken</enum>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="textFormat">
          <enum>Qt::TextFormat::PlainText</enum>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
         </property>
         <property name="textInteractionFlags">
          <set>Qt::TextInteractionFlag::LinksAccessibleByMouse|Qt::TextInteractionFlag::TextEditable|Qt::TextInteractionFlag::TextEditorInteraction|Qt::TextInteractionFlag::TextSelectableByKeyboard|Qt::TextInteractionFlag::TextSelectableByMouse</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
